src/shared/database/entities/user.entity.ts

```diff
@@ -14,4 +15,7 @@ export class UserEntity extends Base {
 
   @Column()
   sessionKey!: string;
+
+  @Column({ default: UserStateEnum.NOT_VERIFIED })
+  state!: UserStateEnum;
```

Comment:
You don't need `!` when you set the default value - it's already guaranteed not to be empty